api/nodes/serializers.py

```diff
@@ -150,13 +151,17 @@ def create(self, validated_data):
         auth = Auth(user)
         node = self.context['view'].get_node()
         pointer_node = Node.load(validated_data['node']['_id'])
-        pointer = node.add_pointer(pointer_node, auth, save=True)
-        return pointer
+        if not pointer_node:
+            raise exceptions.NotFound('Node not found.')
```

Comment:
I think 404 is fine.  Per https://dev.twitter.com/overview/api/response-codes, 404 when "The URI requested is invalid or the resource requested, such as a user, does not exists."  The pointer is not a query parameter.